### ISSUE TITLE
[Ruby 2.7] warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call

### ISSUE THREAD

**Issue by lucascaton** [Hirurg103/capybara_select2#24]
After upgrading to Ruby 2.7, this warning started showing up:

```sh
lib/capybara_select2/helpers.rb:50: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```

**Comment by Hirurg103** [comment]
Hi @lucascaton , 

thank you for reporting the issue. I am currently busy on other things, when I have time I will fix this warning.

If you want to help with improving this project, PRs are welcome!

**Comment by Hirurg103** [comment]
Fixed in #26
